lib/rubocop/cop/rspec/subject_stub.rb

```diff
@@ -74,71 +76,54 @@ class SubjectStub < Cop
         PATTERN
 
         def on_block(node)
+          # process only describe/context/... example groups
           return unless example_group?(node)
 
-          find_subject_stub(node) do |stub|
+          # skip already processed example group
+          # it's processed if is nested in one of the processed example groups
+          return unless (processed_example_groups & node.ancestors).empty?
```

Comment:
Is this the same as `return if processed_example_groups.include?(node.ancestors)`?

Reply:
No, it isn't